### ISSUE TITLE
Exploit - You can pick up Ice and Fire dragons with a Summoning Pendulum.

### ISSUE THREAD

**Issue by ProsperCraft** [lumien231/Random-Things#345]
You can pick up huge dragons that are aggressive with a summoning pendulum.

**Comment by lumien231** [comment]
This is probably an issue on their side. Their Dragons should implement the IMob Interface like the Vanilla Dragon.

**Comment by AlexModGuy** [comment]
No. Implementing IMob would render tamed dragons, which are passive and the same mob, to be attacked by Golems and removed by the game when peaceful, etc, other things IMob entails. You should check if a Mob has the player as a target or not as well in your code. this issue is on your side.

**Comment by ProsperCraft** [comment]
Is it possible to fix this on randomthings side?

**Comment by TheDeviantCrafter** [comment]
Do Ice and Fire dragons class their mobs as bosses? That would prevent it from working with most 'mob in a bottle' items, taking damage from Botania's Bellethorne and quite a few other cases. Random Things could check for that flag.

**Comment by AlexModGuy** [comment]
classing them as bosses would result in the game treating them like bosses. I don't want to be looking at 8 different dragon health bars at my base all day.
